### PR TITLE
CSS Admin Page

### PR DESCRIPTION
**Pull request by avanheeren**
Fixed css for admin page

### CODE REVIEW

File: resources/views/admin.blade.php [hunk 1]
```diff
@@ -57,27 +57,27 @@
                     <div class="card-content">
                         <div class="content">
                             <header class="card-header-title columns">Pending Blogs
-                                <h1 class="padding-left-50">
-                                    <button>
+                                <h1 class="column has-text-right content-margin">
+                                    <button class="button is-light">Add a Blog
```
Comment (Sanyi0001) [inline resources/views/admin.blade.php]: Double add a blog

**Review by Sanyi0001 — CHANGES_REQUESTED** [review]

**Review by awqfrxx — COMMENTED** [review]
Looks good, maybe wait with merging because in my pull request I added 2 more cards in the admin panel, then you can instantly add the css there too?

**Review by awqfrxx — APPROVED** [review]
looks good

**Review by AlexGit123 — APPROVED** [review]
all good

**Review by Sanyi0001 — APPROVED** [review]

**Pull request merged**